src/history/hs.c

```diff
@@ -1268,7 +1327,8 @@ __wt_hs_find_upd(WT_SESSION_IMPL *session, WT_ITEM *key, const char *value_forma
              * update here we fall back to the datastore version. If its timestamp doesn't match our
              * timestamp then we return not found.
              */
-            if ((ret = hs_cursor->next(hs_cursor)) == WT_NOTFOUND) {
+            ret = __wt_hs_cursor_next(session, hs_cursor);
+            if (ret == WT_NOTFOUND) {
```

Comment:
This looks like a `WT_ERR_NOTFOUND_OK(__wt_hs_cursor_next(session, hs_cursor), true)` - is that right?

Reply:
aedc9c4